content/doc/pipeline/index.adoc

```diff
@@ -471,12 +318,14 @@ is really shorthand for:
 
 One way to use tools by default is to add them to your executable path using the special variable env that is defined for all pipelines:
 
-node {
+....
+ node {
```

Comment:
In the future, try to use the following syntax so we get good syntax highlighting when it's rendered:

```asciidoc
[source, groovy]
----
node {
}
----
```